src/main/java/com/auth0/net/StatusCodeRequest.java

```diff
@@ -0,0 +1,38 @@
+package com.auth0.net;
+
+import com.auth0.exception.APIException;
+import com.auth0.exception.Auth0Exception;
+import com.auth0.json.mgmt.jobs.JobError;
+import com.fasterxml.jackson.core.type.TypeReference;
+import okhttp3.OkHttpClient;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+import java.io.IOException;
+
+public class StatusCodeRequest extends EmptyBodyRequest<JobError> {
```

Comment:
I created a new request type and corresponding `JobError` object in order to provide the ability to inspect the HTTP status code returned by the `job_error` API endpoint.  There didn't appear to be any other way to get that info aside from manually mapping the HTTP status code into the field directly as part of the response parsing.